ddl/db_test.go

```diff
@@ -5075,23 +5076,33 @@ func (s *testDBSuite1) TestModifyColumnTime(c *C) {
 		{"year", `99`, "timestamp", "", errno.ErrTruncatedWrongValue},
 	}
 
-	for _, t := range tests {
-		tk.MustExec("drop table if exists t_mc")
-		tk.MustExec(fmt.Sprintf("create table t_mc(a %s)", t.from))
-		tk.MustExec(fmt.Sprintf(`insert into t_mc (a) values (%s)`, t.value))
-		_, err := tk.Exec(fmt.Sprintf(`alter table t_mc modify a %s`, t.to))
-		if t.err != 0 {
-			c.Assert(err, NotNil, Commentf("%+v", t))
-			c.Assert(err, ErrorMatches, fmt.Sprintf(".*[ddl:%d].*", t.err), Commentf("%+v", t))
-			continue
-		}
-		c.Assert(err, IsNil, Commentf("%+v", t))
+	var wg sync.WaitGroup
+	wg.Add(len(tests))
+	for i, t := range tests {
```

Comment:
I don't think it's necessary to drive so many goroutines.

Reply:
I think so :-)

Reply:
I think so :-)